source/includes/mongodb-compatibility-table-scala.rst

```diff
@@ -6,6 +6,7 @@
    :class: compatibility-large
 
    * - Scala Driver Version
+     - MongoDB 6.0
```

Comment:
We already have 6.0 on the table. Is this meant to be 6.1?

Reply:
Good catch!